infrastructure/local/seed.cnw.yml

```diff
@@ -1,14 +1,14 @@
 self:
-  graphqlUrl: http://backend:3001/graphql
-  hostname: 'backend'
+  graphqlUrl: http://cloud-nine-wallet-backend:3001/graphql
+  hostname: 'cloud-nine-wallet-backend'
   mapHostname: 'primary-map'
   openPaymentPublishedPort: 3000
 asset:
   code: USD
   scale: 2
 peers:
   - initialLiquidity: '100000'
-    peerUrl: http://peer-backend:3002
+    peerUrl: http://happy-life-bank-backend:3002
     peerIlpAddress: test.peer
```

Comment:
should we rename `peerIlpAddress` to include something related to `happy-life-bank`?